nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-authorization-providers/src/test/java/org/apache/nifi/authorization/CompositeUserGroupProviderTestBase.java

```diff
@@ -22,6 +22,7 @@
 import org.junit.Assert;
 
 import java.util.HashMap;
```

Comment:
According to the automated build, this import is no longer used and should be removed.

Reply:
Absolutely, I just removed HashMap. Hope the workflow will pass this time!